src/MultiphaseStaggeredHierarchyIntegrator.cpp

```diff
@@ -152,8 +153,9 @@ MultiphaseStaggeredHierarchyIntegrator::MultiphaseStaggeredHierarchyIntegrator(
     d_has_vel_nullspace = input_db->getBoolWithDefault("has_vel_nullspace", d_has_vel_nullspace);
     d_convec_limiter_type = IBAMR::string_to_enum<LimiterType>(
         input_db->getStringWithDefault("convec_limiter_type", IBAMR::enum_to_string(d_convec_limiter_type)));
-    d_convective_time_stepping_type =
-        IBAMR::string_to_enum<TimeSteppingType>(input_db->getStringWithDefault("convec_ts_type", "FORWARD_EULER"));
+    d_convective_time_stepping_type = IBAMR::string_to_enum<IBAMR::TimeSteppingType>(
+        input_db->getStringWithDefault("convec_ts_type", "FORWARD_EULER"));
```

Comment:
note: using IBAMR::TimeSteppingType makes it clear this enum class is from IBAMR